soc/arm/microchip_mec/common/soc_gpio.c

```diff
@@ -0,0 +1,344 @@
+/*
```

Comment:
Just a comment on file naming. Do we need the 'soc_' prefix on these? I think gpio.c would be good enough